### PR TITLE
Add comctl32 support

### PR DESCRIPTION
**Pull request by skdltmxn**
I added support for comctl32

### CODE REVIEW

File: src/prsht.rs [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+// Licensed under the MIT License <LICENSE.md>
+//! Interface for the Windows Property Sheet Pages
+#[repr(C)] #[allow(missing_copy_implementations)]
+pub struct PSP {
+    unused: ::c_void,
+}
+pub type HPROPSHEETPAGE = *mut PSP;
+
+#[repr(C)] #[allow(missing_copy_implementations)]
+pub struct _PROPSHEETPAGEA {
```
Comment (retep998) [inline src/prsht.rs]: Could you instead have this be `PROPSHEETPAGEA` and just have it be a typedef to `PROPSHEETPAGEA_V4`?

**Comment by retep998** [comment]
In https://github.com/skdltmxn/winapi-rs/blob/comctl32/src/commctrl.rs#L149 you have extra parentheses that aren't needed. I'd comment on the diff, but the diff was too big for github to display it. :disappointed:

**Comment by skdltmxn** [comment]
I removed parentheses and _PROPSHEETPAGEA since there is typedef below.

**Comment by retep998** [comment]
Could you rebase this PR, and your other PRs, against master?

**Comment by skdltmxn** [comment]
Ok, it's rebased

**Comment by retep998** [comment]
Your msvc tests appear to be testing functions that don't exist at runtime on appveyor. Could you figure out which symbols they are and disable them from testing?

**Comment by skdltmxn** [comment]
hmm it's strange, I can't find any failure from msvc test logs but only a message saying "Build execution time has reached the maximum allowed time for your plan (40 minutes)."

**Comment by retep998** [comment]
The issue is that if a program links to a symbol which doesn't actually exist in the DLL, then at runtime the process will error. How the error manifests depends on the environment it is run in, and apparently in the appveyor environment it pops up a dialog saying there is a missing symbol. Since it is automated CI and there is nobody to click that dialog it just hangs until it times out.

**Comment by skdltmxn** [comment]
Resolved!

**Pull request merged**